targeteditor/src/main/scala/explore/targeteditor/MagnitudeForm.scala

```diff
@@ -124,15 +126,17 @@ object MagnitudeForm {
                           EnumViewSelect(
                             id = magnitude.band.toString + "_BAND",
                             value = magnitudeView.zoom(Magnitude.band),
-                            exclude = state.usedBands - magnitude.band
+                            exclude = state.usedBands - magnitude.band,
+                            disabled = props.disabled
                           )
                         )
                       ),
                       TableCell(
                         <.span(
                           EnumViewSelect(
                             id = magnitude.band.toString + "_SYSTEM",
-                            value = magnitudeView.zoom(Magnitude.system)
+                            value = magnitudeView.zoom(Magnitude.system),
+                            disabled = props.disabled
```

Comment:
Should all the buttons on the form be disabled, too?

Reply:
Yes, that's a good idea.